include/istio/mixerclient/environment.h

```diff
@@ -28,6 +29,10 @@ namespace mixerclient {
 // Uses UNAVAILABLE status code to indicate network failure.
 using DoneFunc = std::function<void(const ::google::protobuf::util::Status&)>;
```

Comment:
can we move this DoneFunction to "control"

Reply:
As discussed, we will keep DoneFunc in this file.